sql/expression/function/spatial/spatial_reference.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2023 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package spatial
+
+import "github.com/dolthub/go-mysql-server/sql/types"
+
+type SpatialRef struct {
+	Name          string
+	ID            uint32
+	Organization  interface{}
+	OrgCoordsysId interface{}
+	Definition    string
+	Description   interface{}
+}
+
+var SupportedSRIDs = map[uint32]SpatialRef{
```

Comment:
Thinking about how our codebase will change over time... it would be really cool if new SRIDs showed up in information_schema automatically when we add support for them – otherwise, it'll be tough to know to update multiple places in the codebase so they show up here, too. I haven't looked at the implementation details, but having some sort of registry for SRIDs and making them encode any extra metadata needed to display them in `information_schema` is one idea. 

Definitely not needed for this PR, but a follow-up idea to consider.